web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerUtils.java

```diff
@@ -28,7 +28,7 @@ public static boolean checkHttpUrl(String url) {
 
             return 200 == connection.getResponseCode();
         } catch (Exception e) {
-            logger.info(String.format("Error checking link '%s'", url));
+            logger.info(String.format("Error checking link '%s' reason '%s'", url, e));
             logger.debug(e);
```

Comment:
Should get rid of this line so people in debugging mode don't see the error twice